src/SFA.DAS.PSRService.Web/Controllers/ReportController.cs

```diff
@@ -267,6 +295,53 @@ public IActionResult OrganisationName(string post)
             return View("OrganisationName", organisationVm);
         }
 
+        [Route("TotalEmployees")]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult TotalEmployees()
+        {
+            var report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId);
+
+            if (!_reportService.CanBeEdited(report))
+                return new RedirectResult(Url.Action("Index", "Home"));
+
+            var totalEmployeesVm = new TotalEmployees
+            {
+                HasTotalEmployeesMeetMinimum = report.HasTotalEmployeesMeetMinimum,
+                Report = report
+            };
+
+            ViewBag.CurrentPeriod = report?.Period ?? _currentPeriod;
+
+            return View("TotalEmployees", totalEmployeesVm);
+        }
+
+        [Route("PostTotalEmployees")]
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Authorize(Policy = PolicyNames.CanEditReport)]
+        public IActionResult PostTotalEmployees(TotalEmployees totalEmployeesVm)
+        {
+            var totalEmployeesViewModel = new TotalEmployees
+            {
+                Report = _reportService.GetReport(_currentPeriod.PeriodString, EmployerAccount.AccountId)
+            };
+
+            totalEmployeesViewModel.Report.HasTotalEmployeesMeetMinimum =  totalEmployeesVm.HasTotalEmployeesMeetMinimum;
+
+            _reportService.SaveReport(totalEmployeesViewModel.Report, _userService.GetUserModel(User));
+
+            if (totalEmployeesVm.HasTotalEmployeesMeetMinimum == true)
+                return new RedirectResult(Url.Action("Edit", "Report"));
+            else 
+                return new RedirectResult(Url.Action("ValidateEmployeeCount"));
+        }
+
+        [Route("ValidateEmployeeCount")]
```

Comment:
I think this is an odd name for this route. We aren't validating anything here. The response isn't invalid. We are just informing the user that they don't have to submit a report based on the answers they have given